src/orderbooks.md

```diff
@@ -0,0 +1,204 @@
+
+## Data model
+
+You can review the list below to gain a deeper understanding of the data models for each component that can be built using this API:
+
+- [Calls](https://sora-xor.github.io/sora2-network/order_book/pallet/enum.Call.html)
+- [Errors](https://sora-xor.github.io/sora2-network/order_book/pallet/enum.Error.html#variants)
+- [Events](https://sora-xor.github.io/sora2-network/order_book/pallet/enum.Event.html#variants)
+- [Storages](https://sora-xor.github.io/sora2-network/order_book/pallet/index.html#types)
+- [More info](https://sora-xor.github.io/sora2-network/order_book/index.html#reexport.UserOrders)
+
+## Integration Example
+
+Here are code snippets written in multiple languages:
+
+::: code-group
+
+```ts [ts]
+import { api } from "@sora-substrate/util";
+import { PriceVariant } from "@sora-substrate/liquidity-proxy";
+
+import { delay, withConnectedAccount } from "./util";
+
+async function main(): Promise<void> {
+  await withConnectedAccount(async () => {
+    const xor =
+      "0x0200000000000000000000000000000000000000000000000000000000000000";
+    const val =
+      "0x0200040000000000000000000000000000000000000000000000000000000000";
+    const account = "cnVkoGs3rEMqLqY27c2nfVXJRGdzNJk2ns78DcqtppaSRe8qm";
+
+    const orderBooks = await api.orderBook.getOrderBooks();
+    console.log("orderBooks", orderBooks);
+
+    await api.orderBook.getUserOrderBooks(account);
+
+    api.orderBook
+      .subscribeOnUserLimitOrdersIds(val, xor, account)
+      .subscribe((ids) => {
+        console.log("ids", ids);
+      });
+
+    const order = await api.orderBook.getLimitOrder(val, xor, 16);
+    console.log("order", order);
+
+    api.orderBook.subscribeOnAggregatedAsks(val, xor).subscribe((asks) => {
+      console.log("asks", asks);
+    });
+
+    api.orderBook.subscribeOnAggregatedBids(val, xor).subscribe((bids) => {
+      console.log("bids", bids);
+    });
+
+    const price = "1100000000000000000";
+    const amount = "100000000000000000000";
+    const side: PriceVariant = PriceVariant.Buy;
+
+    await api.orderBook.placeLimitOrder(val, xor, price, amount, side);
+
+    await delay(100000);
+  });
+}
+
+main()
+  .catch(console.error)
+  .finally(() => process.exit());
+```
+
+:::
+
+These code samples demonstrate various functionalities, from managing storage subscriptions to executing extrinsics.
+
+## Step-by-step instructions
+
+In this section, you will learn how to perform various operations related to order books on the SORA blockchain in `polkadot.js` (or dotapps.io) starting in the `Developer` -> `Extrinsics` tab. Whether you want to create a new order book, delete an existing one, update its details, change the status, place a limit order, or cancel a limit order, this guide will provide you with clear and concise instructions.
```

Comment:
```suggestion
In this section, you will learn how to perform various operations related to order books on the SORA blockchain in `polkadot.js` (or dotapps.io) starting in the `Developer` -> `Extrinsics` tab. Whether you want to create a new order book, delete an existing one, update its details, change the status, this guide will provide you with clear and concise instructions.
```

Most users won't use polkadot.js to open an order. We should explain in the wiki how users can propose to open a new trading pair (create a new order book), delete, or update its details. Need to mention that everything is done through governance and subject to community vote. Redirect to fast tracking a proposal for faster governance process.

@bgodlin